SPECS/SymCrypt-OpenSSL/SymCrypt-OpenSSL.spec

```
@@ -63,6 +66,10 @@ install SymCryptProvider/symcrypt_prov.cnf %{buildroot}%{_sysconfdir}/pki/tls/sy
 %{_sysconfdir}/pki/tls/symcrypt_prov.cnf
 
 %changelog
+* Thu Apr 25 2023 Maxwell Moyer-McKee <mamckee@microsoft.com> - 1.4.2-1
+- Support additional parameters in the SymCrypt provider required for TLS connections
+- Various bugfixes for TLS scenarios
+
 * Wed Apr 17 2023 Maxwell Moyer-McKee <mamckee@microsoft.com> - 1.4.1-1
```

Since you're in here...
```suggestion
* Wed Apr 17 2024 Maxwell Moyer-McKee <mamckee@microsoft.com> - 1.4.1-1
```